container-build/cerc-watcher-erc20/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:16.17.1-alpine3.16
+
+RUN apk --update --no-cache add git python3 alpine-sdk
+
+WORKDIR /app
+
+COPY . .
+
+RUN echo "Building watcher-ts" && \
```

Comment:
Can we follow the model used in the other containers and not pull source in Dockerfile?
(convention is that source is cloned in `laconic-so setup-repositories` and in the `laconic-so build-containers` phase it uses the repo cloned already).